addons/account/models/account_move.py

```diff
@@ -546,10 +546,16 @@ def _sequence_fixed_regex(self):
         groups="account.group_account_invoice,account.group_account_readonly",
     )
 
+    # ==== Hash Fields ====
+    restrict_mode_hash_table = fields.Boolean(related='journal_id.restrict_mode_hash_table')
+    secure_sequence_number = fields.Integer(string="Inalteralbility No Gap Sequence #", readonly=True, copy=False)
+    inalterable_hash = fields.Char(string="Inalterability Hash", readonly=True, copy=False)
+    string_to_hash = fields.Char(compute='_compute_string_to_hash', readonly=True)
+
+    # We need the btree index for unicity constraint (`_check_unique_sequence_number`) AND this one for human searches
```

Comment:
Surely the first part of the comment should be near the corresponding field, and only the second half should be near the `create_index`?

And the comment should be near the `create_index` itself since the override creates 3 different indexes, 2 of which have no relation whatsoever to the comment?